homework/03LessonThree.js

```diff
@@ -110,10 +112,14 @@ var threeByThreeArray = [
 
 // Problem 11:
 // Replace the center element of threeByThreeArray, which is 5, with something else.
+threeByThreeArray[2,1] = 1;
 
```

Comment:
Hey Matt! We may need to revisit this and see why this is passing. the index of 5 is actually [1][1]